src/components/about.tsx

```diff
@@ -0,0 +1,17 @@
+interface AboutProps {
+  name: string;
+  description: string;
+}
+
+const About = ({ name, description }: AboutProps) => {
+  return (
+    <div className="flex flex-col items-center text-center font-Malayalam font-normal">
```

Comment:
```suggestion
    <div className="flex flex-col items-center text-center font-Malayalam">
```
not needed, does nothing